graylog2-server/src/test/java/org/graylog2/alerts/FormattedEmailAlertSenderTest.java

```diff
@@ -49,10 +50,11 @@
     private NodeId mockNodeId;
 
     private FormattedEmailAlertSender emailAlertSender;
+    private final Engine templateEngine = new Engine();
 
     @Before
     public void setUp() throws Exception {
-        this.emailAlertSender = new FormattedEmailAlertSender(new EmailConfiguration(), mockNotificationService, mockNodeId);
+        this.emailAlertSender = new FormattedEmailAlertSender(new EmailConfiguration(), mockNotificationService, mockNodeId, templateEngine);
```

Comment:
Why is it not possible to mock out the engine instead of supplying a real instance?